src/components/widgets/SelectWidget.js

```diff
@@ -23,15 +23,30 @@ function SelectWidget({
   value,
   defaultValue,
   required,
+  multiple,
   onChange
 }) {
   return (
     <select
+      multiple={multiple}
       title={placeholder}
       value={value}
       defaultValue={defaultValue}
       onChange={(event) => {
-        onChange(processValue(schema.type, event.target.value));
+        let newValue;
+        if (multiple) {
+          newValue = [];
+          const options = event.target.options;
+          for (let i = 0; i < options.length; i++) {
+            if (options[i].selected) {
+              newValue.push(options[i].value);
+            }
```

Comment:
Note: If it's matter of it not fully implementing the Array interface, you can simply `[].call(options).map(...)`.